stake-pool/cli/src/main.rs

```diff
@@ -567,11 +589,12 @@ fn add_associated_token_account(
     account
 }
 
-fn command_deposit(
+fn command_deposit_stake(
     config: &Config,
     stake_pool_address: &Pubkey,
     stake: &Pubkey,
     token_receiver: &Option<Pubkey>,
+    referrer: &Option<Pubkey>,
```

Comment:
nit: just to disambiguate, can we call this `referrer_token_account`?  Just this interface makes it seem like it will derive an ATA from this pubkey

Reply:
good point. In general, referrer is a fixed entity, else the token receiver. Perhaps `token_receiver` should also be renamed `pool_token_receiver_account`.

Reply:
Yes please, great idea!